### PR TITLE
[MM-47383] Make MFA enforcement a community feature

### PR DESCRIPTION
**Pull request by plant99**
#### Summary

This PR removes licence checks during MFA actions related to enforcement, thus making it available to every deployment(self / free-cloud).

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-47383
#### Related Pull Requests

- Has server changes (https://github.com/mattermost/mattermost-server/pull/21451)
- Has mobile changes (please link here)

#### Release Note

```release-note
Made MFA enforcement a community feature.
```

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -5,7 +5,6 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "mattermost-webapp",
```
Comment (M-ZubairAhmed) [inline package-lock.json:8]: please revert this change

Comment (plant99) [reply package-lock.json]: thanks for catching this!

**Review by M-ZubairAhmed — APPROVED** [review]

**Comment by plant99** [comment]
Closing this, please see JIRA for details.
